src/index.js

```diff
@@ -50,7 +71,7 @@ function startQtumProcess(reindex) {
     flags.push('-reindex');
   }
 
-  const qtumProcess = spawn(qtumdPath, flags);
+  qtumProcess = spawn(qtumdPath, flags, { detached: true });
```

Comment:
in detached mode?
```
On Windows, setting options.detached to true makes it possible for the child process to continue running after the parent exits. The child will have its own console window. Once enabled for a child process, it cannot be disabled.

On non-Windows platforms, if options.detached is set to true, the child process will be made the leader of a new process group and session. Note that child processes may continue running after the parent exits regardless of whether they are detached or not.
```

Reply:
I believe I had some issues with running it in attached mode. I will test again.

Reply:
Just tested, looks ok to run in attached mode. Must have fixed it forgot to remove the flag.